pp/integrations/plugin-packs/procedures/cloud-aws-elasticache.md

```diff
@@ -2,55 +2,316 @@
+| ElastiCache-Items       | Cloud-Aws-ElastiCache-Items-Api              | Check items number             | X       |
+| ElastiCache-Network     | Cloud-Aws-ElastiCache-Network-Api            | Check network usage            | X       |
+| ElastiCache-Replication | Cloud-Aws-ElastiCache-Replication-Api        | Check replication performances | X       |
+| ElastiCache-Requests    | Cloud-Aws-ElastiCache-Requests-Memcached-Api | Check Memcached performances   | X       |
+| ElastiCache-Requests    | Cloud-Aws-ElastiCache-Requests-Redis-Api     | Check Redis performances       | X       |
+| ElastiCache-Usage       | Cloud-Aws-ElastiCache-Usage-Memcached-Api    | Check Memcached space usage    | X       |
+| ElastiCache-Usage       | Cloud-Aws-ElastiCache-Usage-Redis-Api        | Check Redis space usage        | X       |
+
+### Discovery rules
+
+The pack provides a discovery rule to automatically discover Elasticache resources:
+
+![image](../../../assets/integrations/plugin-packs/procedures/cloud-aws-elasticache-provider.png)
+
+More information about the Host Discovery module is available in the Centreon documentation: [Host Discovery](/docs/monitoring/discovery/hosts-discovery)
+
+### Collected metrics & status
+<Tabs groupId="sync">
+<TabItem value="ElastiCache-Commands" label="ElastiCache-Commands">
+
+| Metric name      | Description                                                                            |
+| :--------------- | :------------------------------------------------------------------------------------- |
+| CPUUtilization   | The percentage of CPU utilization. Unit: Percent                                       |
+
+</TabItem>
+<TabItem value="ElastiCache-Connections" label="ElastiCache-Connections">
+
+| Metric name      | Description                                                                            |
+| :--------------- | :------------------------------------------------------------------------------------- |
+| CurrConnections  | A count of the number of connections connected to the cache at an instant in time. ElastiCache uses two to three of the connections to monitor the cluster.In addition to the above, memcached creates a number of internal connections equal to twice the threads used for the node type. The thread count for the various node types can be see in the Nodetype Specific Parameters of the applicable Parameter Group. Unit: Count                                                                                                 |
+| NewConnections   |The number of new connections the cache has received. This is derived from the memcached total_connections statistic by recording the change in total_connections across a period of time. This will always be at least 1, due to a connection reserved for a ElastiCache. Unit: Count                                                                                               |
```

Comment:
```suggestion
| NewConnections   | The number of new connections the cache has received. This is derived from the memcached total_connections statistic by recording the change in total_connections across a period of time. This will always be at least 1, due to a connection reserved for ElastiCache. Unit: Count                                                                                               |
```